assets/src/js/admin/reports/app/index.js

```diff
@@ -1,42 +1,39 @@
 // Reports page app
 
 // Vendor dependencies
-import { createContext } from 'react'
-import { Switch, Route } from "react-router-dom"
-import moment from 'moment'
+import moment from 'moment';
 
 // Store related dependencies
-import { StoreProvider } from './store'
-import { reducer } from './store/reducer'
+import { StoreProvider } from '../store';
+import { reducer } from '../store/reducer';
 
 // Components
-import PeriodSelector from '../components/period-selector'
-import Tabs from '../components/tabs'
-import Routes from '../components/routes'
+import PeriodSelector from '../components/period-selector';
+import Tabs from '../components/tabs';
+import Routes from '../components/routes';
 
-const App = (props) => {
+const App = () => {
+	// Initial app state (available in component through useStoreValue)
+	const initialState = {
+		// Initial period range (defaults to the past week)
+		period: {
+			startDate: moment().subtract( 7, 'days' ),
+			endDate: moment(),
+			range: 'week',
+		},
+	};
 
-    // Initial app state (available in component through useStoreValue)
-    const initialState = {
-        // Initial period range (defaults to the past week)
-        period: { 
-            startDate: moment().subtract(7, 'days'),
-            endDate: moment(),
-            range: 'week'
-        }
-    }
-
-    return (
-        <StoreProvider initialState={initialState} reducer={reducer}>
-            <div className='wrap give-settings-page'>
-                <div className='give-settings-header'>
-                    <h1 className='wp-heading-inline'>Reports</h1>
-                    <PeriodSelector />
-                </div>
-                <Tabs />
-                <Routes />
-            </div>
-        </StoreProvider>
-    )
-}
-export default App
+	return (
+		<StoreProvider initialState={ initialState } reducer={ reducer }>
+			<div className="wrap give-settings-page">
+				<div className="give-settings-header">
+					<h1 className="wp-heading-inline">Reports</h1>
```

Comment:
@henryholtgeerts can we localize this string?

Reply:
Change made, ref: https://github.com/impress-org/givewp/pull/4424/commits/97c63ca856fad2bddfb3497be85d6a5bd3a31e0e